api/prisma/seed-staging.ts

```diff
@@ -972,13 +972,13 @@ export const stagingSeed = async (
           minOccupancy: 1,
           monthlyRent: '1200',
           numBathrooms: 1,
-          numBedrooms: 1,
+          numBedrooms: 0,
           number: '101',
           sqFeet: '750.00',
           amiChart: { connect: { id: amiChart.id } },
           unitTypes: {
             connect: {
-              id: unitTypes[1].id,
+              id: unitTypes[5].id,
```

Comment:
Curious why this is 5 and not 0 when it is 0 above when numBedrooms is 0?

Reply:
SRO is different than Studio

Reply:
Yeah, the order of IDs in the enum is sort of weird. SRO is after several of the multi-bedroom types.